### PR TITLE
refactoring game1 to add to APIUrl and APIFunctions

### PR DESCRIPTION
**Pull request by kishore223**

### CODE REVIEW

**Comment by mepripri** [comment]
I have run the code in my local everything seems super fine. It's satisfying to see codes getting refactored sharing same files and being into their own folders.:) Great work. But i see a little change in the progress bar, its taking 101% due to Math.ceil(). Can you please try parseInt() and check that once. I have tried from my end.

File: Front-End/src/components/Game1/Quiz.js [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+      <div className="padding-main">
+        <div className="button-contianer">
+          <BsXCircleFill
+            className="btn-color"
+            onClick={(e) => {
+              direct(e);
+            }}
+          />
+          <ProgressBar
+            now={Math.ceil((count + 1) * (100 / cardNo))}
```
Comment (mepripri) [inline Front-End/src/components/Game1/Quiz.js:61]: This is the place where i see the change. Can you please update it. Thanks :)

Comment (kishore223) [reply Front-End/src/components/Game1/Quiz.js]: I have Updated. Can you please check it.

**Comment by mepripri** [comment]
Yep the change is fine. Approving and Signing it off.

**Pull request merged**